client/src/components/dataset/DatasetTable.jsx

```diff
@@ -114,136 +54,118 @@ export default class DatasetTable extends Component {
   }
 
   getCellClassName(columnTitle) {
-    if (this.state.activeSidebar &&
-      this.state.activeSidebar.columnTitle === columnTitle) {
+    const { sidebarProps } = this.state;
+    if (sidebarProps != null && sidebarProps.columnTitle === columnTitle) {
```

Comment:
👍 I'll fix this. Also, we should compare columnName instead of title since we now have that property available